Jenkinsfile

```diff
@@ -101,7 +101,13 @@ pipeline {
       cleanupAndNotify(currentBuild.currentResult)
     }
     unsuccessful {
-      cleanupAndNotify(currentBuild.currentResult, "#development", "@secrets-provider-for-k8s-owners")
+        script {
+            if (env.BRANCH_NAME == 'master') {
+              cleanupAndNotify(currentBuild.currentResult, "#development", "@secrets-provider-for-k8s-owners")
+            } else {
+              cleanupAndNotify(currentBuild.currentResult, "#development", "")
```

Comment:
this can be `cleanupAndNotify(currentBuild.currentResult, "#development")` as `""` is the dedault value for the additional message, as can be seen in the implementation for [cleanupAndNotify](https://github.com/conjurinc/jenkins-pipeline-library/blob/master/vars/cleanupAndNotify.groovy).

You can technically also omit the `#development` but i'd leave it for explicitness.